update.py

```diff
@@ -750,8 +751,29 @@ def copy_static_html_sites(site, destdir):
             pass
 
 
-#######################################################################
+def check_imports():
+    '''check key imports work'''
+    import pkg_resources
+    requires = ["sphinx_rtd_theme>=0.1.8", "sphinxcontrib.vimeo>=0.0.1"]
```

Comment:
sphinxcontrib.vimeo does not exist. sphinxcontrib-vimeo does. However, neither are installed by the setup script any more. See #4195.